docs/RouterTutorial/react-navigation-tab-kr.md

```diff
@@ -1,414 +1,677 @@
+import HomeScreen from './screens/HomeScreen';
+import ChatScreen from './screens/ChatScreen';
+import SettingsScreen from './screens/SettingsScreen';
+
+const TabNavigator = createBottomTabNavigator(
+  {
+    Home: {
+      screen: HomeScreen,
+    },
+    Chat: {
+      screen: ChatScreen,
+    },
+    Settings: {
+      screen: SettingsScreen,
+    },
+  },
+  {
+    defaultNavigationOptions: ({navigation}) => ({
+      tabBarIcon: ({horizontal, tintColor}) => {
+        const {routeName} = navigation.state;
+        let iconName;
+        if (routeName === 'Home') {
+          iconName = 'ios-home';
+        } else if (routeName === 'Chat') {
+          iconName = 'ios-chatboxes';
+        } else if (routeName === 'Settings') {
+          iconName = 'ios-settings';
+        }
+        return (
+          <Ionicons
+            name={iconName}
+            size={horizontal ? 20 : 25}
+            color={tintColor}
+          />
+        );
+      },
+    }),
+  },
+);
+export default createAppContainer(TabNavigator);
+
+```
+
+
+
+- **아이콘에 뱃지 추가**
+
+  <img src="./images/Tab_iconwithbadge.png" style="zoom:25%;" />
+
+아이콘에 뱃지가 필요한 경우 뷰 컨테이너를 추가하여 아이콘 방식으로 추가시켜줍니다.
+
+(아래 예제는 UI 상에서만 뱃지를 추가한 상태입니다.  redux, mobx 등을 통하여 상태관리를 해주어야합니다.)
+
+*IconWithBadge.js*
+
+```jsx
+import React from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+
+export default class IconWithBadge extends React.Component {
+  render() {
+    const {name, badgeCount, color, size} = this.props;
+    return (
+      <View style={styles.container}>
+        <Ionicons name={name} size={size} color={color} />
+        {badgeCount > 0 && (
+          <View style={styles.badge}>
+            <Text style={styles.text}>{badgeCount}</Text>
+          </View>
+        )}
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  container: {
+    width: 24,
+    height: 24,
+    margin: 5,
+  },
+  badge: {
+    position: 'absolute',
+    right: -6,
+    top: -3,
+    backgroundColor: 'red',
+    borderRadius: 6,
+    width: 12,
+    height: 12,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  text: {
+    color: 'white',
+    fontSize: 10,
+    fontWeight: 'bold',
+  },
+});
+
+```
+
+
+
+### 6. NavigationConfig
+
+이제 기본적인 기능 이외에도 config를 사용하여 부가적인 요소들을 설정해 Tab을 완성시켜보겠습니다.
+
+앞서 기본 라이브러리들을 설명하면서 bottomTabNavigator, MaterialBottomTabNavigator, MaterialTopTabNavigator 에서 스타일 부분에서 차이가 생긴다고 했는데 바로 이 config부분에서 각각 몇가지의 다른 기능을 제공하기 때문입니다! 
+
+이번 챕터에서는 각각의 라이브러리에서  config를 사용하여 예제를 만들고 몇가지 기능들을 소개해보겠습니다.
+
+[React Navigation 공식 문서]: https://reactnavigation.org/docs/en/bottom-tab-navigator.html
+
+(React Navigation 공식 문서를 참조하였고, 더 많은 config기능들을 공식 문서에서 확인하실 수 있습니다.)
+
+
+
+###### createBottomTabNavigator
+
+createBottomTabNavigator을 사용한 예제에서는 activetintcolor와 inactivetintcolor을 사용해 라벨과 아이콘의 활성화 여부에 따른 색깔을 설정해주었고 style에서 backgroundcolor을 사용해 tab bar의 배경색을 지정해주었습니다. 
```

Comment:
`activetintcolor` -> `activeTintColor`
`inactivetinitcolor` -> `inactivetinitcolor`
`backgroundcolor` -> `backGroundColor`
(for consistency)